mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -164,3 +170,162 @@ def _convert_mm_to_m(array):
+
+    .. versionadded:: 1.7
+    """
+    _validate_type(inst, (BaseRaw, BaseEpochs, Evoked), "inst")
+    _validate_type(calibration, Calibration, "calibration")
+    _check_option("to", to, ("radians", "pixels"))
+    _check_calibration(calibration)
+
+    # get screen parameters
+    screen_size = calibration["screen_size"]
+    screen_resolution = calibration["screen_resolution"]
+    dist = calibration["screen_distance"]
+
+    # loop through channels and convert units
+    converted_chs = []
+    for ch_dict in inst.info["chs"]:
+        if ch_dict["coil_type"] == FIFF.FIFFV_COIL_EYETRACK_POS:
+            unit = ch_dict["unit"]
+            name = ch_dict["ch_name"]
+
+            if ch_dict["loc"][4] == -1:  # x-coordinate
+                size = screen_size[0]
+                res = screen_resolution[0]
+            elif ch_dict["loc"][4] == 1:  # y-coordinate
+                size = screen_size[1]
+                res = screen_resolution[1]
+            else:
+                raise ValueError(
+                    f"loc array not set properly for channel '{name}'. Index 4 should"
+                    f"  be -1 or 1, but got {ch_dict['loc'][4]}"
+                )
+            # check unit, convert, and set new unit
+            if to == "radians":
+                if unit != FIFF.FIFF_UNIT_PX:
+                    raise ValueError(
+                        f"Data must be in pixels in order to convert to radians."
+                        f" Got {unit} for {name}"
+                    )
+                inst.apply_function(
+                    _pix_to_rad, picks=name, size=size, res=res, dist=dist
+                )
+                ch_dict["unit"] = FIFF.FIFF_UNIT_RAD
+            elif to == "pixels":
+                if unit != FIFF.FIFF_UNIT_RAD:
+                    raise ValueError(
+                        f"Data must be in radians in order to convert to pixels."
+                        f" Got {unit} for {name}"
+                    )
+                inst.apply_function(
+                    _rad_to_pix, picks=name, size=size, res=res, dist=dist
+                )
+                ch_dict["unit"] = FIFF.FIFF_UNIT_PX
+            converted_chs.append(name)
+    if converted_chs:
+        logger.info(f"Converted {converted_chs} to {to}.")
+        if to == "radians":
+            # check if any values are greaater than .44 radians
+            # (25 degrees) and warn user
+            data = inst.get_data(picks=converted_chs)
+            if np.any(np.abs(data) > 0.52):
+                warn(
+                    "Some visual angle values subtend greater than .52 radians "
+                    "(30 degrees), meaning that the conversion between pixels "
+                    "and visual angle may be very non-linear. Take caution when "
+                    "interpreting these values. Max visual angle value in data:"
+                    f" {np.round(np.nanmax(data), 2)} radians.",
```

Comment:
Can just use float formatting for this
```suggestion
                    f" {np.nanmax(data):0.2f} radians.",
```